README.md

```diff
@@ -381,10 +381,10 @@ Other Style Guides
 
     ```javascript
     // bad
-    const bar = [...foo].map(bar);
```

Comment:
why do these need to change?

Reply:
You're declaring `bar`. If you're declaring it here, how can it be what you pass in to `map`?

Reply:
If this line of code works as it's written, I am more terrified of JavaScript than ever before

Reply:
aha, true :-)